tabular/tests/unittests/models/test_image_predictor.py

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Can we have a similar script in `examples/`, so it can be shared with folks that want to test out this functionality?

Reply:
Would prefer to get a golden dataset example for multimodality, we currently don't have one besides petfinder and petfinder is somewhat large for an example.